fragments/build-wrapper/accelerator.yaml

```diff
@@ -0,0 +1,18 @@
+accelerator:
+  displayName: Include build tool wrapper
+
+  options:
+    - name: includeBuildToolWrapper
+      inputType: checkbox
+      label: Include Build Tool wrapper support
+      dataType: boolean
+      defaultValue: true
+
+engine:
+  merge:
+    - condition: "#includeBuildToolWrapper AND #buildTool == 'maven'"
```

Comment:
Since `buildTool` is not defined in the fragment  it should not be used in the fragment. 

In practice, using a variable/option that is defined in the importing accelerator may work, but it has inconsistent behaviour and is not actually a supported feature.

Reply:
It's a tough one, because this could be considered a (semi) "private" option that we don't want exposed to accelerators, but is still a way to communicate with the caller. Then it still needs to be defined in some contract somehow.

Currently, the only exposed option is `includeBuildToolWrapper`. Why this one and not `buildTool`? I think a solution to this particular issue here could be to expose `buildTool` AND to have special value `none` (all as a Choice). This still limits the number of exposed options (if this was a concern) and retains full behavior

Reply:
The `buildTool` option is only relevant when the accelerators provides a choice between maven and gradle. For any other accelerators they have to pass in the build tool that they use. I wonder if there is a way the fragment could define any required options in addition to the ones defined in the fragment.